src/layouts/core-dev-dot-layout/core-dev-dot-layout.module.css

```diff
@@ -1,7 +1,13 @@
 .root {
   /* Widths/spacing needed in multiple stylesheets */
   --dev-dot-sidebar-width: 320px;
-  --main-area-padding-left-right: 24px;
+
+  /* Note: "right" value is not used in multiple stylesheets,
+     so is declared in sidebar-sidecar-layout.module.css */
+  --main-area-padding-left: 24px;
```

Comment:
As noted in this comment, `--main-area-padding-left` is shared across components, but there's no need to share `--main-area-padding-right`, so I have kept it local to `sidebar-sidecar-layout`.